src/users/users.service.ts

```diff
@@ -23,6 +25,12 @@ export class UsersService {
   ) {}
 
   async create(createUserDto: CreateUserDto, strategy: ICreateStrategy) {
+    // Validate ReCaptcha
+    const isHuman = await this.validateReCaptcha(createUserDto.reCaptchaToken);
+    if (!isHuman) {
```

Comment:
Should we rather throw a ForbiddenException instead? This will resolve into a 500 Internal Server Error.

Reply:
I fully agree with you. Great catch! 🚀